contextual_lenses/pfam_family_classification/pfam_utils.py

```diff
@@ -80,11 +81,29 @@ def create_pfam_df(family_accessions, test=False):
   pfam_df['mod_family_accession'] = pfam_df.family_accession.apply(lambda x: mod_family_accession(x))
   pfam_df = pfam_df[pfam_df.mod_family_accession.isin(family_accessions)]
   pfam_df['index'] = pfam_df.family_id.apply(lambda x: family_id_to_index[x])
-  pfam_df['one_hot_inds'] = pfam_df.sequence.apply(lambda x: residues_to_one_hot_inds(pad_seq(x)))
+
+  pfam_df['one_hot_inds'] = pfam_df.sequence.apply(lambda x: residues_to_one_hot_inds('M' + x[:511])) # x[:512] # pad_seq(x)
```

Comment:
can you add a comment explaining the motivation for this?

Reply:
oh yes, the 'M' + x[:511] was for prepending an 'M', I've removed this for now but we may want to add it back later